features/machine/machine.feature

```diff
@@ -141,6 +141,98 @@ Feature: Machine features testing
     Given I scale the machineset to +2
     Then the machineset should have expected number of running machines
 
+  # @author zhsun@redhat.com
+  @admin
+  @destructive
+  Scenario Outline: Machineset should have relevant annotations to support scale from/to zero
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
```

Comment:
You still need the followings because when you delete the machineset after scenario, the removal of nodes is not immediate, thus have to wait for a clean teardown in order not to break its following tests.
```
Given I store the number of machines in the :num_to_restore clipboard
And admin ensures node number is restored to "<%= cb.num_to_restore %>" after scenario
```